setup.cfg

```diff
@@ -18,31 +18,8 @@ known_first_party=aiokafka,kafka
 known_third_party=pytest
 
 [mypy]
-check_untyped_defs = True
-disallow_any_generics = True
-disallow_untyped_defs = True
-follow_imports = silent
-strict_optional = True
-warn_redundant_casts = True
-warn_unused_ignores = True
-warn_unused_configs = True
-
-[mypy-pytest]
-ignore_missing_imports = True
-
-[mypy-Cython]
-ignore_missing_imports = True
-
-[mypy-kafka.*]
-ignore_missing_imports = True
-warn_no_return = False
-disallow_untyped_defs = False
-
-[mypy-kafka.vendor.*]
-ignore_missing_imports = True
-
-[mypy-aiokafka.*]
-ignore_missing_imports = True
-
-[mypy-aiokafka.util]
-ignore_missing_imports = False
+python_version = 3.7
```

Comment:
python 3.7 has already [been deprecated](https://github.com/aio-libs/aiokafka/releases/tag/v0.8.1) in the latest version